src/main/resources/db/migration/V010__ADD_PP_INDEXES.sql

```diff
@@ -0,0 +1,4 @@
+CREATE INDEX idx_company_name ON payment_position (company_name);
```

Comment:
`company_name` is rarely used, consider whether to keep it and to change the default sorting logic (now is based on `company_name`)
cc: @aferracci @pagopa/pagopa-team-core